pl.art.lach.slawek.apps.v2blankbrowser.json

```diff
@@ -0,0 +1,28 @@
+{
+  "id": "pl.art.lach.slawek.apps.V2BlankBrowser",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "40",
+  "sdk": "org.gnome.Sdk",
+  "command": "V2BlankBrowser",
+  "clear": [ "*.o"],
+  "modules": [
+  {
+    "name": "main",
```

Comment:
usuallly the module has a name related to what the actually module is. not "main"